react/src/components/Widget/Widget.tsx

```diff
@@ -186,11 +183,7 @@ export const Widget: React.FC<WidgetProps> = props => {
   const query: string[] = [];
   const isMissingWidgetContainer = !totalReceived;
   const addressDetails = useAddressDetails(to, isMissingWidgetContainer);
-  const isFiat: boolean =
-    currency !== 'SAT' &&
-    currency !== 'BCH' &&
-    currency !== 'bits' &&
-    currency !== 'XEC';
+  const isFiat: boolean = currency !== 'BCH' && currency !== 'XEC';
```

Comment:
`isFiat` seems like a core function. Might deserve to be in a currency service.
Also I'd expect this to look more like `fiatCurrencies.includes(currency)`. Especially if we end up adding another cryptocurrency.

Reply:
I'm not quite sure how to do `fiatCurrencies.includes(currency)` the typescript way yet. The possible currencies themselves are types. Hm. Maybe by importing `isFiat` and type checking it from the utils folder. Alright I have some ideas for this

Reply:
In the next stacked branch I moved `isFiat` to `util/api-client.ts` which is functionally serving as a currency service